src/AppBundle/Bridge/Chameleon/Migration/Script/update-1554708763.inc.php

```diff
@@ -0,0 +1,8 @@
+<h1>Build #1554708763</h1>
+<h2>Date: 2019-04-08</h2>
+<div class="changelog">
+    - Remove module_cms_search table
+</div>
+<?php
+
+TCMSLogChange::deleteTable('module_cms_search');
```

Comment:
This isn't used anywhere? (Bundle-Updates?)

Reply:
Nope. It is also only removed in chameleon-system, so projects won't be affected.